.github/workflows/node.js.yml

```diff
@@ -27,3 +27,4 @@ jobs:
           echo ::add-matcher::.github/eslint.json
           yarn run eslint . --format=compact
       - run: yarn test
+      - run: yarn readme:check
```

Comment:
could we run readme:check as one of the tests? this way we won't forget locally then be reminded by CI?

Reply:
good call -- done!